test/e2e/run-openrpc-api-test-coverage.ts

```diff
@@ -0,0 +1,418 @@
+        {
+          name: 'blockNumberExample',
+          description: 'Example of a blockNumber request',
+          params: [],
+          result: {
+            name: 'blockNumberResult',
+            value: '0x1',
+          },
+        },
+      ];
+
+      const personalSign = openrpcDocument.methods.find(
+        (m) => (m as MethodObject).name === 'personal_sign',
+      );
+
+      (personalSign as MethodObject).examples = [
+        {
+          name: 'personalSignExample',
+          description: 'Example of a personalSign request',
+          params: [
+            {
+              name: 'data',
+              value: '0xdeadbeef',
+            },
+            {
+              name: 'address',
+              value: ACCOUNT_1,
+            },
+          ],
+          result: {
+            name: 'personalSignResult',
+            value: '0x1a8819e0c9bab700',
+          },
+        },
+      ];
+
+      const switchEthereumChain = openrpcDocument.methods.find(
+        (m) => (m as MethodObject).name === 'wallet_switchEthereumChain',
+      );
+      (switchEthereumChain as MethodObject).examples = [
+        {
+          name: 'wallet_switchEthereumChain',
+          description:
+            'Example of a wallet_switchEthereumChain request to sepolia',
+          params: [
+            {
+              name: 'SwitchEthereumChainParameter',
+              value: {
+                chainId: '0xaa36a7',
+              },
+            },
+          ],
+          result: {
+            name: 'wallet_switchEthereumChain',
+            value: null,
+          },
+        },
+      ];
+
+      const signTypedData4 = openrpcDocument.methods.find(
+        (m) => (m as MethodObject).name === 'eth_signTypedData_v4',
+      );
+
+      const signTypedData4Example = (signTypedData4 as MethodObject)
+        .examples?.[0] as ExamplePairingObject;
+
+      // just update address for signTypedData
+      (signTypedData4Example.params[0] as ExampleObject).value = ACCOUNT_1;
+
+      // update chainId for signTypedData
+      (
+        signTypedData4Example.params[1] as ExampleObject
+      ).value.domain.chainId = 1337;
+
+      // net_version missing from execution-apis. see here: https://github.com/ethereum/execution-apis/issues/540
+      const netVersion: MethodObject = {
+        name: 'net_version',
+        summary: 'Returns the current network ID.',
+        params: [],
+        result: {
+          description: 'Returns the current network ID.',
+          name: 'net_version',
+          schema: {
+            type: 'string',
+          },
+        },
+        description: 'Returns the current network ID.',
+        examples: [
+          {
+            name: 'net_version',
+            description: 'Example of a net_version request',
+            params: [],
+            result: {
+              name: 'net_version',
+              description: 'The current network ID',
+              value: '0x1',
```

Comment:
does the server return 0x1 for net_version even though its 1337 for chainId?

Reply:
net_version is network version not the chainId. for 99.99999% of chains the networkId is going to be `1`

Reply:
oh right I was thinking of networkId